python/scripts/bash/run-docker-dev.sh

```
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker run --rm -it -v $(pwd):/home polymarket-agents:latest bash
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)